cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/README.md

```diff
@@ -0,0 +1,277 @@
+# Multiple AKS Clusters Deployment with Azure Traffic Manager
```

Comment:
The readme should follow the format of other environments like https://github.com/microsoft/bedrock/tree/master/cluster/environments/azure-single-keyvault specifically, redundant information should not be present in the readme.

Reply:
@ReenuSaluja  , take a look at the branch in your forked repo: https://github.com/ReenuSaluja/bedrock/tree/bedrockpull-sg/cluster/environments/azure-multiple-clusters-waf-tm-apimgmt

Reply:
Resolved and build test run successfully